README.md

```diff
@@ -1,191 +1,249 @@
+```bash
+poetry run pypanther list rules --log-types Slack.AuditLogs --default-severity HIGH
+```
+```
++----------------------------------------------------+-----------------+------------------+---------+
+|                         id                         |    log_types    | default_severity | enabled |
++----------------------------------------------------+-----------------+------------------+---------+
+|       Slack.AuditLogs.DLPModified-prototype        | Slack.AuditLogs |       HIGH       |   True  |
+|     Slack.AuditLogs.EKMConfigChanged-prototype     | Slack.AuditLogs |       HIGH       |   True  |
+|  Slack.AuditLogs.EKMSlackbotUnenrolled-prototype   | Slack.AuditLogs |       HIGH       |   True  |
+| Slack.AuditLogs.IDPConfigurationChanged-prototype  | Slack.AuditLogs |       HIGH       |   True  |
+| Slack.AuditLogs.LegalHoldPolicyModified-prototype  | Slack.AuditLogs |       HIGH       |   True  |
+|    Slack.AuditLogs.MFASettingsChanged-prototype    | Slack.AuditLogs |       HIGH       |   True  |
+| Slack.AuditLogs.PrivateChannelMadePublic-prototype | Slack.AuditLogs |       HIGH       |   True  |
+|    Slack.AuditLogs.SSOSettingsChanged-prototype    | Slack.AuditLogs |       HIGH       |   True  |
+| Slack.AuditLogs.UserPrivilegeEscalation-prototype  | Slack.AuditLogs |       HIGH       |   True  |
++----------------------------------------------------+-----------------+------------------+---------+
+Total rules: 9
+```
 
-- **`content/overrides/`**: The `overrides/` directory is dedicated for managing your overrides to built-in rules. We recommend defining new rule override functions (like title or severity), attribute overrides (like include_filters), and mass-updates using for loops using the `apply_overrides()` function. Check the `content/rules` folder for an example.
+### Inspecting Rules
+View the source code and configuration of a specific rule:
+```bash
+poetry run pypanther get rule Slack.AuditLogs.MFASettingsChanged-prototype
+```
+```
+class SlackAuditLogsMFASettingsChanged:
+    create_alert = True
+    dedup_period_minutes = 60
+    display_name = Slack MFA Settings Changed
+    enabled = True
+    log_types = ['Slack.AuditLogs']
+    id = Slack.AuditLogs.MFASettingsChanged-prototype
+    summary_attributes = ['p_any_ip_addresses', 'p_any_emails']
+    threshold = 1
+    tags = ['Slack', 'Defense Evasion', 'Modify Authentication Process', 'Multi-Factor Authentication']
+    reports = {'MITRE ATT&CK': ['TA0005:T1556.006']}
+    default_severity = HIGH
+    default_description = Detects changes to Multi-Factor Authentication requirements
+    default_destinations = None
+    default_reference = https://slack.com/intl/en-gb/help/articles/204509068-Set-up-two-factor-authentication
+    rule = 
+    def rule(self, event):
+        return event.get("action") == "pref.two_factor_auth_changed"
+    ...
+```
 
-### Setting Your Configuration
+### Testing Rules
+Run tests on a specific rule with detailed output:
```

Comment:
What about "Run tests locally on a specific rule..."? I want to add the `local` so people are not surprised when the tests fail when ran on Panther side (could be due to missing dependencies and what not)

Reply:
Can you give context on when that scenario would occur?

Reply:
1. 3rd party dependency present locally but not present in Panther
2. Tests that rely on some resources that are present locally but not supporting in Panther (I believe customer ran into this because recently because they were reading some json files from their rules and doing stuff with them